2021/2021-01/Submissions/AlbertHenry/game.py

```diff
@@ -1 +1,36 @@
-""" Test submissions """
+from random import choice
+
+opts = dict(R = "Rock", P = "Paper", S = "Scissors")
+
+win_against = dict(R = "S", P = "R", S = "P")
+
+def who_win(opt1, opt2):
+    if opt1 == opt2:
+        result = None
+    elif win_against[opt1] == opt2:
+        result = opt1
+    else:
+        result = opt2
+    return result
```

Comment:
this function can probably just return the result string that you print to screen in the game loop. Worth renaming opt1 and opt2 to make clear that one is the player and one is the computer choice